### ISSUE TITLE
Questions about validation process

### ISSUE THREAD

**Issue by r09522817** [Open-Debin/Emotion-FAN#28]
Hi, I found that you guys split the video into 3 segments in training set to train. However, in validation set, you guys only choose the first image and validate on it. Just wondering why don't split the video into 3 segments in validation set and validate it. Thank you!
